src/components/TodoComponents/TodoForm.js

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import { FaPlus } from 'react-icons/fa';
 
-const TodoForm = props => {
+const TodoForm = ({ inputHandler, addHandler, enterKeyHandler, text }) => {
   return (
     <div className="todo-input">
       <input
         placeholder="What do you want to do?"
+        onChange={event => inputHandler(event)}
+        onKeyPress={(event) => enterKeyHandler(event)}
+        value={text}
       />
-      <button><FaPlus /></button>
+      <button onClick={() => addHandler()}>
+        <FaPlus />
+      </button>
     </div>
-  )
-}
+  );
+};
 
 export default TodoForm;
```

Comment:
- Great  job on the assignment Noble. 
- You are properly handling state and all the events in React. 
- Your design looks really good, nice way of using  react-icons 💯  good job with this one 👍
- You split Your app into smaller components to make them modular. You can add todos succesfully , there is not long until your todo app is fully functional 💃 
- Your git workflow is really fluid and detailed, keep up the good job with this.